readme.txt

```diff
@@ -151,6 +151,9 @@ Yes, Custom Fonts is completely free to use, without any limitation.
 
 
 == Changelog ==
+= 2.1.4 =
```

Comment:
This file's stable tag version needs to be update on L-7 @yssambare12

Reply:
Updated.